tests/modules/test_text_processor.py

```diff
@@ -1,54 +1,162 @@
+import mock
+from mock import patch
+from datetime import datetime
 from django.test import TestCase
+
 from modules.text_processor import TextProcessor
-from modules.i18n import hindi_remind, hindi_information
+from modules.i18n import hindi_remind, hindi_information, msg_placeholder_child, \
+                         msg_subscribe, msg_unsubscribe, msg_failure, msg_failed_date
 
 class TextProcessorGetDataTests(TestCase):
     def test_all_caps(self):
         t = TextProcessor()
         keyword, child_name, date = t.get_data_from_message("REMIND NATHAN 25/11/2015")
         self.assertEqual(keyword, "remind")
         self.assertEqual(child_name, "nathan")
-        self.assertEqual(date, "25/11/2015")
+        self.assertEqual(date, datetime(2015, 11, 25, 0, 0))
```

Comment:
In the model, dates are stored as Dates and not DateTime objects, so while this will work in the test, it may be best to test these against Date objects.